runtime/JavaScript/src/antlr4/Lexer.d.ts

```diff
@@ -22,7 +22,9 @@ export declare class Lexer extends Recognizer<number> {
     nextToken(): Token;
     skip(): void;
     more(): void;
-    more(m: number): void;
+    mode(m: number): void;
```

Comment:
can you add a setMode both here and in js and mark this one as deprecated ?

Reply:
What will be the next ANTLR version, i.e. from which version should it be marked as deprecated?
since 4.13.2?